wao/notifier.py

```diff
@@ -2,14 +2,30 @@
 
 import requests
 import sys
+import time
+from wao._429_file_util import delete_429_file, write_to_429_file
+
 sys.path.append(EXECUTION_PATH)
 from config import Config as ExecutionConfig
 
 
 def send_start_deliminator_message(brain, coin, month, year, dup, max_counter_dup):
     print("notifier: send_start_deliminator_message: ")
-    text = "========" + str(brain) + " DUP = " + str(dup) + " MAX_COUNTER_DUP = " + max_counter_dup + " " + str(coin) + " " + str(month) + " " + str(year) + "=======>"
-    if ExecutionConfig.NOTIFIER_ENABLE:
-        result = requests.post('https://api.telegram.org/bot' + ExecutionConfig.NOTIFIER_TELEGRAM_BOT_API_TOKEN_BACKTEST +
-                                   '/sendMessage?chat_id=' + ExecutionConfig.NOTIFIER_TELEGRAM_CHANNEL_ID_BACKTEST + '&text=' + text + '&parse_mode=Markdown')
-        print(str(result))
\ No newline at end of file
+    text = "========" + str(brain) + " DUP = " + str(dup) + " MAX_COUNTER_DUP = " + max_counter_dup + " " + str(
+        coin) + " " + str(month) + " " + str(year) + "=======>"
+
+    post_request(text)
+
+
+def post_request(text, is_from_429_watcher=False):
+    print("post_request: " + text)
+    result = requests.post('https://api.telegram.org/bot' + ExecutionConfig.NOTIFIER_TELEGRAM_BOT_API_TOKEN_429 +
+                           '/sendMessage?chat_id=' + ExecutionConfig.NOTIFIER_TELEGRAM_CHANNEL_ID_BACKTEST + '&text=' + text + '&parse_mode=Markdown')
+    print(str(result))
+    if str(result) != "<Response [200]>":
```

Comment:
Handle 429 only explicitly

Reply:
here 429 doesn't happen other errors are the once happening when message overlap happens that is why I am using this you raised this on the previous PR as well and I gave you this reply and you said to handle it using else: but that creates further problems so this is the most efficient way I found

Reply:
ok let's remove the else